test/verify/files/mock-insights

```diff
@@ -137,7 +138,9 @@ class handler(BaseHTTPRequestHandler):
 
 
 def insights_server(port):
-    HTTPServer(('', port), handler).serve_forever()
+    httpd = HTTPServer(('', port), handler)
```

Comment:
I got this far :)

Reply:
Some random dude wrote [this blog post](https://www.piware.de/2011/01/creating-an-https-server-in-python/) a decade ago, which was useful :grin: